rollup.config.js

```diff
@@ -34,6 +35,11 @@ const postcssPlugins = [
     browsers: browserslist.production,
     autoprefixer: { grid: true },
   }),
+  // we need to place the postcssDiscardComments BEFORE postcssCustomProperties,
+  // otherwise we will end up with a bunch of empty :root elements
+  // wherever there are imported comments
+  // see https://github.com/postcss/postcss-custom-properties/issues/123
```

Comment:
this comment is important and I tried to explain it's relevance in the PR description.

Reply:
Thanks for the comment! I'd also add a link to this PR to the comment as the PR's description does such a nice job of explaining what's going on.